src/ng/compile.js

```diff
@@ -1386,6 +1386,61 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
     return TTL;
   };
 
+  var ignoreMDirectives = false;
```

Comment:
what about {{disableCommentDirectives}} or {{commentDirectivesDisabled}}?
or do you prefer {{commentDirectivesEnabled}} ?